src/app/ReviewService.Abstractions/IReviewPrompter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// Provides ways to prompt user to review the current application.
+/// </summary>
+/// <remarks>
+/// On iOS, you can't submit a review while developing, but the review prompt will be shown in your simulator or device.
+///
+/// On Android, you can't see the review prompt while developing or if you distribute the application manually.
+/// You have to download the application from Google Play Store to see the review prompt.
+///
+/// See https://github.com/jamesmontemagno/StoreReviewPlugin for more details.
+/// </remarks>
+public interface IReviewPrompter
+{
+	/// <summary>
+	/// Prompts the user to rate the current application using the platform's default application store.
+	/// </summary>
+	/// <returns><see cref="Task"/>.</returns>
+	Task TryPrompt();
```

Comment:
Shouldn't we have a `ct` passed here?

Reply:
The package we use to display the prompt doesn't take any CancellationToken for his async method.

Reply:
I figured that we could still manually cancel it by using `ct.Register` https://learn.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken.register?view=net-7.0. But it may be overkill

Reply:
Could do it.